build.gradle

```diff
@@ -140,7 +141,7 @@ allprojects {
 
 
   apply plugin: 'org.gradle.java-library'
-  sourceCompatibility = 11
+  sourceCompatibility = 8
```

Comment:
This is about what language features are available + what Java version is supposed to be recorded in classfiles (44 + 8 = 52.0, 44 + 11 = 55.0)